bayesian/repr_license_finder_go.py

```diff
@@ -0,0 +1,114 @@
+"""Identifies representative license for go eco repo by considering all the dependencies."""
+
+from __future__ import division
+from flask import current_app
+from requests import get, post, exceptions
+import json
+from .utils import GREMLIN_SERVER_URL_REST
+from f8a_worker.utils import get_session_retry
+from .utils import fetch_file_from_github
+import requests
+import logging
+import traceback
+
+
+class GetReprLicense():
+    """Summary.
+
+    input:url:repo url to find repr_license
+    ecosystem:ecosystem of the repo
+    output:repr_license.
+    """
+
+    def __init__(self, url, ecosystem, license_api):
+        """Get all the inputs required to the class."""
+        self.url = url
+        self.ecosystem = ecosystem
+        self.license_api = license_api
+
+    def fetch_go_deps_from_github(self):
```

Comment:
@vasulakkaraju you can use the `fetch_file_from_github` function from `utils` I don't think you need to define this function again.

Reply:
yes the same function is used anyhow ill remove this definition from here @ravsa Thanks for your comment.